src/Microsoft.AspNetCore.Authentication.Abstractions/ChallengeContext.cs

```diff
@@ -26,20 +26,13 @@ public ChallengeContext(HttpContext httpContext, string authenticationScheme)
         /// <param name="httpContext">The context.</param>
         /// <param name="authenticationScheme">The name of the scheme.</param>
         /// <param name="properties">The properties.</param>
-        /// <param name="behavior">The challenge behavior.</param>
-        public ChallengeContext(HttpContext httpContext, string authenticationScheme, AuthenticationProperties properties, ChallengeBehavior behavior)
+        public ChallengeContext(HttpContext httpContext, string authenticationScheme, AuthenticationProperties properties)
             : base(httpContext, authenticationScheme, properties)
         {
             if (string.IsNullOrEmpty(authenticationScheme))
```

Comment:
You should consider removing this, as it's a check already enforced by the base context class.